services/ui-src/src/verbiage/pages/accordion.ts

```diff
@@ -112,47 +112,34 @@ export default {
       intro: [
         {
           type: "text",
-          as: "header",
-          content: "<strong>State or Territory User Instructions<strong>",
+          as: "span",
+          content: "<b>State or Territory User instructions</b>",
         },
         {
           type: "text",
           as: "span",
           content:
-            "<br>This reporting tool is to be used by grantees for reporting of MFP program data. The information provided in this report will allow CMS to monitor grantee progress and identify challenges and improvement opportunities. For additional guidance on completing this form, please see the associated ",
+            "<br><br>Click “Add new MFP SAR submission” below to begin reporting for the current reporting period. For additional guidance on completing this form, see the associated ",
         },
         {
           type: "externalLink",
-          content: "User Guide",
+          content: "User Guide and Help File",
           props: {
-            href: "https://www.ecfr.gov/current/title-42/chapter-IV/subchapter-C/part-438/subpart-A/section-438.8#p-438.8(k)", // TO-DO: currently placeholder, replace when correct link available
+            href: "https://www.google.com",
```

Comment:
switching back to a placeholder?

Reply:
I don't actively remember making this change tbh, but the old link was also a placeholder so either works!